src/end.sol

```diff
@@ -193,24 +193,24 @@ contract End is LibNote {
     }
 
     // --- Data ---
-    VatLike  public vat;
+    VatLike  public vat;   // CDP Engine
     CatLike  public cat;
-    VowLike  public vow;
+    VowLike  public vow;   // Debt Engine
     PotLike  public pot;
     Spotty   public spot;
 
-    uint256  public live;  // cage flag
-    uint256  public when;  // time of cage
-    uint256  public wait;  // processing cooldown length
-    uint256  public debt;  // total outstanding dai following processing [rad]
+    uint256  public live;  // Active Flag
+    uint256  public when;  // Time of cage                   [unix epoch time]
```

Comment:
FYI the way unit labels are aligned is not consistent throughout the PR...here we right-justify, other places we left-justify, and there's even one place that's neither (mid-justified, maybe?). If there is some aesthetic intention in each case then NBD, but thought I'd point it out.